src/main/java/io/cryostat/net/web/http/api/v1/TargetRecordingPatchStop.java

```diff
@@ -79,6 +75,9 @@ void handle(RoutingContext ctx, ConnectionDescriptor connectionDescriptor) throw
                                     .findFirst();
                     if (descriptor.isPresent()) {
                         connection.getService().stop(descriptor.get());
+                        recordingTargetHelper.cancelScheduledNotificationIfExists(recordingName);
+                        recordingTargetHelper.notifyRecordingStopped(
+                                recordingName, connectionDescriptor.getTargetId());
```

Comment:
What if the `connection.getService().stop(desc)` call was extracted out and refactored into a method like `RecordingTargetHelper#stopRecording()`? We already have start/delete/get in there, so it would be a nice cleanup. I think it would also be nice to avoid having `notifyRecordingStopped` and `cancelScheduledNotificationIfExists` as public methods - those should be private and called by the helper whenever it makes sense to do so, not called by other callsites like HTTP handlers.